invenio_app_ils/schemas/documents/document-v1.0.0.json

```diff
@@ -160,19 +123,466 @@
+            "uniqueItems": true
+          },
+          "alternative_names": {
+            "items": {
+              "minLength": 1,
+              "type": "string"
+            },
+            "minItems": 1,
+            "type": "array",
+            "uniqueItems": true
+          },
+          "email": {
+            "format": "email",
+            "minLength": 1,
+            "type": "string"
+          },
+          "full_name": {
+            "minLength": 1,
+            "title": "Author name",
+            "type": "string"
+          },
+          "roles": {
+            "items": {
+              "default": "author",
+              "enum": ["author", "editor", "illustrator"],
+              "minLength": 1,
+              "type": "string"
+            },
+            "minItems": 1,
+            "type": "array",
+            "uniqueItems": true
+          }
+        },
+        "required": ["full_name"],
+        "type": "object"
+      },
+      "minItems": 1,
+      "title": "Authors",
+      "type": "array",
+      "uniqueItems": false
+    },
+    "copyrights": {
+      "items": {
+        "additionalProperties": false,
+        "properties": {
+          "holder": {
+            "minLength": 1,
+            "title": "Copyright holder",
+            "type": "string"
+          },
+          "material": {
+            "enum": [
+              "addendum",
+              "additional material",
+              "data",
+              "e-book",
+              "e-proceedings",
+              "editorial note",
+              "erratum",
+              "preprint",
+              "publication",
+              "reprint",
+              "software",
+              "translation"
+            ],
+            "minLength": 1,
+            "title": "Material to which the copyright refers",
+            "type": "string"
+          },
+          "statement": {
+            "minLength": 1,
+            "title": "Copyright notice",
+            "type": "string"
+          },
+          "url": {
+            "format": "uri",
+            "minLength": 1,
+            "title": "Copyright notice URL",
+            "type": "string"
+          },
+          "year": {
+            "type": "integer"
+          }
+        },
+        "type": "object"
+      },
+      "minItems": 1,
+      "title": "List of copyrights to documents in this record",
+      "type": "array",
+      "uniqueItems": true
+    },
+    "corporate_authors": {
+      "items": {
+        "minLength": 1,
+        "title": "List of corporate authors",
+        "type": "string"
+      },
+      "minItems": 1,
+      "type": "array",
+      "uniqueItems": true
+    },
+    "curated": {
+      "type": "boolean"
+    },
+    "document_type": {
       "type": "string",
-      "description": "Document title"
+      "enum": ["BOOK", "PROCEEDINGS", "STANDARD"],
+      "default": "BOOK",
+      "title": "Document type"
     },
-    "_access": {
-      "type": "object",
+    "edition": {
```

Comment:
Is it possible for a document to have multiple editions?

Reply:
I've changed it for only one